nmostesting/NMOSTesting.py

```diff
@@ -832,7 +832,7 @@ def check_internal_requirements():
 
 
 def check_external_requirements():
-    deps = {"sdpoker": ("sdpoker --version", "0.1.0"), "testssl": ("testssl/testssl.sh -v", "3.0rc5")}
+    deps = {"sdpoker": ("sdpoker --version", "0.2.0"), "testssl": ("testssl/testssl.sh -v", "3.0rc5")}

```

Comment:
Micro-suggestion: format this on multiple lines to make it easier to edit independently?
```suggestion
    deps = {
        "sdpoker": ("sdpoker --version", "0.2.0"),
        "testssl": ("testssl/testssl.sh -v", "3.0rc5")
    }
```